lib/Subscribers/SubscriberSegmentRepository.php

```diff
@@ -12,4 +15,77 @@ class SubscriberSegmentRepository extends Repository {
   protected function getEntityClassName() {
     return SubscriberSegmentEntity::class;
   }
+
+  public function getNonDefaultSubscribedSegments(int $subscriberId): array {
+    $qb = $this->entityManager->createQueryBuilder();
+    return $qb->select('ss')
+      ->from(SubscriberSegmentEntity::class, 'ss')
+      ->join('ss.segment', 'seg', Join::WITH, $qb->expr()->eq('seg.type', ':typeDefault'))
```

Comment:
Sorry, I probably don't understand this, but the name of this method suggests it should return all segments except the default one. But here you are only joining the default segment. What am I missing?

Reply:
You are right. I wrote the incorrect condition in the join. I will fix it.